package com.wnet.desafio3.service.exceptions;
//n√£o exige que tenha Try Catch (RunTimeException)
public class ResourceNotFoundException extends RuntimeException{
    public ResourceNotFoundException(String message) {
        super(message);
    }
}
